src/ontology/maxo-edit.owl

```diff
@@ -13002,6 +13022,170 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <h
+AnnotationAssertion(dce:creator <http://purl.obolibrary.org/obo/MAXO_0020011> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dce:date <http://purl.obolibrary.org/obo/MAXO_0020011> "2025-06-01T10:58:42Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/MAXO_0020011> "PMID:30726000")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/MAXO_0020011> "anthropometric assessment")
+SubClassOf(<http://purl.obolibrary.org/obo/MAXO_0020011> <http://purl.obolibrary.org/obo/MAXO_0000527>)
+
+# Class: <http://purl.obolibrary.org/obo/MAXO_0020012> (platelet aggregometry)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/MAXO_0020012> "A diagnostic test that measures how well platelets clump (aggregate) in response to a stimulaus. Light transmittance platelet aggregometry (LTA) in platelet‐rich plasma measures changes in light transmission as platelets aggregate. Impedance aggregometry measures changes in electrical impedance as platelets aggregate.")
+AnnotationAssertion(dce:creator <http://purl.obolibrary.org/obo/MAXO_0020012> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dce:date <http://purl.obolibrary.org/obo/MAXO_0020012> "2025-06-01T11:06:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/MAXO_0020012> "PMID:35510959")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/MAXO_0020012> "platelet aggregometry")
+SubClassOf(<http://purl.obolibrary.org/obo/MAXO_0020012> <http://purl.obolibrary.org/obo/MAXO_0035122>)
+
+# Class: <http://purl.obolibrary.org/obo/MAXO_0020013> (erythrocyte metabolite measurement)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/MAXO_0020013> "A quantitative measurement of metabolites present in the cytoplasm of erythrocytes.")
+AnnotationAssertion(dce:creator <http://purl.obolibrary.org/obo/MAXO_0020013> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dce:date <http://purl.obolibrary.org/obo/MAXO_0020013> "2025-06-01T11:10:45Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/MAXO_0020013> "erythrocyte metabolite measurement")
+SubClassOf(<http://purl.obolibrary.org/obo/MAXO_0020013> <http://purl.obolibrary.org/obo/MAXO_0000829>)
+
+# Class: <http://purl.obolibrary.org/obo/MAXO_0020014> (ventilation/perfusion lung scan)
+
+AnnotationAssertion(dce:creator <http://purl.obolibrary.org/obo/MAXO_0020014> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dce:date <http://purl.obolibrary.org/obo/MAXO_0020014> "2025-06-01T11:17:01Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/MAXO_0020014> "A type of scintigraphy and medical isotopes to evaluate the circulation of air and blood within a patient's lungs, in order to determine the ventilation/perfusion ratio.")
```

Comment:
Wrong property, should be IAO:0000115.

Reply:
I guess this means my Protege settings are wrong? Not sure I understand what to do